animations/base_clip.py

```diff
@@ -0,0 +1,92 @@
+from moviepy.editor import *
+
+
+class BaseClip:
```

Comment:
I'm not sure if this is the way to go; I've been looking into the clips classes and how to join them, and the idea seems to be that all the base clips (videoFile, image, text, etc.) all eventually go into a CompositeVideoClip in some order.

This could be a way to black box all base clip types and apply common effects to them.